asyncpg/transaction.py

```diff
@@ -234,7 +234,7 @@ def __repr__(self):
         attrs = []
         attrs.append('state:{}'.format(self._state.name.lower()))
 
-        attrs.append(self._isolation)
+        attrs.append(str(self._isolation))
```

Comment:
I think `if self._isolation is not None` would be more appropriate here.

Reply:
Ah, right, that is indeed more consistent with the rest of the repr.